lib/core/src/server/config/webpack.config.default.js

```diff
@@ -32,6 +35,20 @@ export function createDefaultWebpackConfig(storybookBaseConfig) {
             },
           ],
         },
+        {
+          test: cssModuleRegex,
+          loaders: [
+            require.resolve('style-loader'),
+            {
+              loader: require.resolve('css-loader'),
+              options: {
+                importLoaders: 1,
+                modules: true,
+                localIdentName: '[name]__[local]___[hash:base64:5]',
+              },
+            },
+          ],
```

Comment:
we'll still want our default postcss in action here.

You could save line 23-34 as `defaultPostcssLoader` variable and then spread it into both style loading blobs.

Reply:
Good catch! CRA has a [`getStyleLoaders`](https://github.com/facebook/create-react-app/blob/f89385f7438f39750b762590ebd9729534e30190/packages/react-scripts/config/webpack.config.dev.js#L45) function and I missed the `postcss-loader` portion of it.

Reply:
That is why we need to integrate with cra properly, and not mimicking it.

Reply:
Agreed, integrating is the better approach. This is just following how it's currently implemented.